src/Controls/src/Core/WebView/WebView.cs

```
@@ -295,6 +295,8 @@ private static string EscapeJsString(string js)
 			if (js == null)
 				return null;
 
+			js = Regex.Replace(js, @"\r\n|\r|\n", " ");
```

Happens in all the platforms?

---

The issue affects Windows (scripts fail silently) and iOS (throws exceptions) but not Android (works correctly by default as its WebView already handles newlines properly).